typescript-test-samples/typescript-sample-app/test/helpers/env.ts

```diff
@@ -0,0 +1,17 @@
+import { beforeAll, afterAll } from '@jest/globals';
+
+let env: any;
+
+beforeAll(() => {
+  jest.resetModules();
```

Comment:
This (`jest.resetModules()`) looks like a side effect of using this helper to allow environment variable mocking. I'm a little concerned that it will surprise folks when they pull this helper in. Should it go somewhere else? Maybe the test fixtures themselves?